### PR TITLE
delete dynamic hash set/get is not supported on any of target now

### PR DESCRIPTION
**Pull request by weiqiand**

### CODE REVIEW

File: tdi_python/tdiTable.py [hunk 1]
```diff
@@ -1908,11 +1908,12 @@ def set_supported_attributes_to_supported_commands(self):
         attributes_dic = {
                   0: ["symmetric_mode_set", "symmetric_mode_get"],
                   2: ["idle_table_set_poll", "idle_table_set_notify", "idle_table_get"],
-                  3: ["dynamic_hash_set", "dynamic_hash_get"],
-                  4: ["meter_byte_count_adjust_set", "meter_byte_count_adjust_get"],
-                  5: ["port_status_notif_cb_set"],
-                  6: ["port_stats_poll_intv_set", "port_stats_poll_intv_get"],
-                  8: ["selector_table_update_cb_set"]}
+                  # dynamic hash set is not supported on any of target now
+                  #3: ["dynamic_hash_set", "dynamic_hash_get"],
```
Comment (saynb) [inline tdi_python/tdiTable.py:1912]: let's just remove

**Review by saynb — APPROVED** [review]

File: tdi_python/tdiTable.py [hunk 2]
```diff
@@ -1908,11 +1908,10 @@ def set_supported_attributes_to_supported_commands(self):
         attributes_dic = {
                   0: ["symmetric_mode_set", "symmetric_mode_get"],
                   2: ["idle_table_set_poll", "idle_table_set_notify", "idle_table_get"],
-                  3: ["dynamic_hash_set", "dynamic_hash_get"],
-                  4: ["meter_byte_count_adjust_set", "meter_byte_count_adjust_get"],
-                  5: ["port_status_notif_cb_set"],
-                  6: ["port_stats_poll_intv_set", "port_stats_poll_intv_get"],
-                  8: ["selector_table_update_cb_set"]}
+                  3: ["meter_byte_count_adjust_set", "meter_byte_count_adjust_get"],
```
Comment (saynb) [inline tdi_python/tdiTable.py:1911]: What is 1?

Comment (weiqiand) [reply tdi_python/tdiTable.py:1911]: {"DynamicKeyMask", TDI_RT_ATTRIBUTES_TYPE_DYNAMIC_KEY_MASK},

Comment (saynb) [reply tdi_python/tdiTable.py:1911]: I wonder why that isn't there. It is supported

**Pull request merged**